sensirion_arch_config.h

```diff
@@ -35,6 +35,20 @@
 #define SPS30_USE_ALT_I2C
 #endif /* __AVR__ */
 
+/*
+ * ESP8266 core < 2.5.0 has a 32 byte I2C receive buffer in the Wire library
+ * Limit readout to PM mass concentrations
+ */
+#ifdef ESP8266
+#include <core_version.h>
+#if defined(ARDUINO_ESP8266_RELEASE_2_3_0) || \
+    defined(ARDUINO_ESP8266_RELEASE_2_4_0) || \
+    defined(ARDUINO_ESP8266_RELEASE_2_4_1) || \
+    defined(ARDUINO_ESP8266_RELEASE_2_4_2)
+#define SPS30_USE_I2C_32_BYTE_LIMIT
```

Comment:
IMHO the wording is a bit unfortunate. I associate `use` more with a feature and limit with an anti-feature.. maybe `#define LIMITED_I2C_BUFFER_SIZE` ?

Reply:
Agreed, that sounds better